### ISSUE TITLE
WordPress Images

### ISSUE THREAD

**Issue by CHRISEVANS1601** [workeffortwaste/horseman#127]
Hi All,

Not really an issue as such - but is anyone aware of a setting that I may need to change to make the crawler scan WP content uploads? I am trying to get it to hook up with ChatGPT to create missing ALT texts - but cant seem to get the crawler to scan the WP Content folder?
